packages/fiori/src/themes/ShellBar.css

```diff
@@ -372,6 +369,7 @@ ui5-icon[data-count]::before {
 	background: var(--sapShell_Hover_Background);
 }
 
+.ui5-shellbar-logo:focus,
```

Comment:
You can't mix :host or ::slotted selectors with others using comma due to IE, please split and duplicate them.

Reply:
Done